sdcm/cluster_k8s/__init__.py

```diff
@@ -1188,6 +1188,19 @@ def gather_k8s_logs(self) -> None:
                                      f"{logfile}-previous.log",
                                      namespace=namespace, ignore_status=True)
 
+                        # NOTE: pick up Scylla container-specific files
+                        if namespace != SCYLLA_NAMESPACE or container_name != 'scylla':
+                            continue
+                        scylla_container_files_to_copy = (
+                            ('/var/lib/scylla/io_properties.yaml', logfile / 'io_properties.yaml'),
+                            ('/etc/scylla.d/', logfile / 'etc-scylla-d'),
+                            ('/etc/scylla/', logfile / 'etc-scylla'),
+                        )
+                        for src_path, dst_path in scylla_container_files_to_copy:
+                            self.kubectl(
+                                f"cp {res}:{src_path} {dst_path} -c {container_name}",
+                                namespace=namespace, ignore_status=True)
```

Comment:
Does it create `{logfile}` directory automatically?

Reply:
Yes, it does. This code was tested and it works as expected -> creates catalog of files which is referenced as a source.